mxcubeweb/routes/ra.py

```diff
@@ -42,7 +42,9 @@ def handle_timeout_gives_control(sid, timeout=30):
                 msg = "Another user is already asking for control"
                 return make_response(msg, 409)
 
+        data = request.get_json()
         current_user.requests_control = True
+        current_user.requests_control_msg = data["message"]
```

Comment:
Storing the requester's message in the corresponding user object in the DB so that it gets sent to the user in control (inside the requester's `observer` object)